### ISSUE TITLE
Search doesn't recognize some items

### ISSUE THREAD

**Issue by Shadowex3** [tom5454/Toms-Storage#136]
I have absolutely no idea what, how, or why but certain random items and not others don't work with search. Chimerite Gems from Mana and Artifice for example will never show up when searching through a Crafting Terminal, but other items like the Animated Quill will. Polished Andesite Stairs from base minecraft will show up, but plain old Andesite won't.

**Comment by Dankey19** [comment]
yeah i have the same problem. Really hope theres an answer because its really annoying and makes it unusable
